utils/segmentclipper/src/imp.rs

```diff
@@ -279,6 +279,14 @@ impl BaseTransformImpl for SegmentClipper {
 
         Ok(())
     }
+
+    fn propose_allocation(
+        &self,
+        _a: Option<&gst::query::Allocation>,
+        _b: &mut gst::query::Allocation,
+    ) -> Result<(), gst::LoggableError> {
+        Ok(())
```

Comment:
Right, but we do not propose any allocation here :-)

Reply:
Also... I am not sure it is a problem for our use case, I think it is generally "wrong" :)

Reply:
I just wanted to kill the error, does it matter that we implement the function? I looked at changing it in gstreamer-rs code, but it seemed like it would be a harder thing to get in. I saw a comment that said you *should* implement this if you touch the buffer in some way.

Reply:
https://gitlab.freedesktop.org/gstreamer/gstreamer-rs/-/merge_requests/1665